torch/testing/_internal/common_nn.py

```diff
@@ -5046,8 +5042,6 @@ def test_cuda(self, test_case, dtype=None, extra_args=None):
         def convert_dtype(obj, dtype, requires_grad=False):
             if isinstance(obj, torch.Tensor):
                 return obj.detach().to(dtype=dtype).requires_grad_(requires_grad)
-            elif isinstance(obj, torch.Tensor):
```

Comment:
Interesting if statement.

Reply:
you have to be sure!

Reply:
my guess is this used to be something like Variable and it got auto changed.

Reply:
That would make sense indeed.